indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,23 @@
         }
     }
 }
+
+#assign-role-table {
```

Comment:
Almost everything is based on a more 'core' component though. The `#assign-role-table` for example is technically a `.i-table-widget`. So in this example, a new "mode" for the `.i-table-widget` could be created. Unfortunately, it's not always easy to come up with a new mode, especially when what you want to change in the core component is a minor thing.